indico/modules/rb_new/client/js/common/timeline/TimelineItem.jsx

```diff
@@ -193,6 +196,14 @@ class TimelineItem extends React.Component {
                     {!!rejectionReason && <div>{Translate.string('Reason: {reason}', {reason: rejectionReason})}</div>}
                 </div>
             );
+        } else if (type === 'conflicting-candidates') {
+            popupContent = (
+                <div styleName="popup-center">
+                    <strong>
+                        <Translate>Candidate is conflicting with other occurrences</Translate>
```

Comment:
Didn't we want to avoid the word "Candidate"? Maybe we can use "Occurrence", same as in the legend.
An idea: "This occurrence is conflicting with an existing booking"